.github/dependabot.yml

```diff
@@ -1,3 +1,19 @@
+# Copyright The Enterprise Contract Contributors
```

Comment:
This should go after the `---` line. Otherwise, this file is interpreted as two yaml documents which may not always work as intended.

Reply:
Perhaps, I'm sure we don't want to have a license header per-document though...